plugins/kubernetes-cluster/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@backstage/plugin-kubernetes-cluster",
```

Comment:
does it make sense to extract this into a different plugin or can we ship it out of `plugin-kubernetes` or `plugin-kubernetes-react`?

Reply:
I think we should create another plugin because it's a different view with different users admins vs service owners